### ISSUE TITLE
Set endpoint in Environment variable

### ISSUE THREAD

**Issue by crypticro** [lostintime/aws-s3-webdav#2]
Is it possible to set an S3 endpoint as e environment variable? Couldn't find any documentation about that.

**Comment by lostintime** [comment]
There is an example running docker in README:

```
docker run --read-only --rm \
    -p 127.0.0.1:8080:8080 \
    -v ~/.aws/credentials:/.aws/credentials \
    -e "AWS_SHARED_CREDENTIALS_FILE=/.aws/credentials" \
    -e "AWS_PROFILE=default" \
    -e "AWS_REGION=eu-central-1" \
    -e "AWS_BUCKET=my-bucket" \
    -e "AWS_KEY_PREFIX=tmp/" \
    lostintime/aws-s3-webdav:0.1.0
```

All the available options passed as env variables her, is this what you need? (source: https://github.com/lostintime/aws-s3-webdav/blob/master/src/main.rs#L27)

**Comment by sloonz** [comment]
Pretty sure endpoint refers to `http://s3.{}.amazonaws.com`, so it can be used with alternate object storages like minio/scaleway…

**Comment by lostintime** [comment]
> Pretty sure endpoint refers to `http://s3.{}.amazonaws.com`, so it can be used with alternate object storages like minio/scaleway…

right, this part is hardcoded, needs to be extracted to an env variable
